### ISSUE TITLE
Improved keyboard navigation in forms

### ISSUE THREAD

**Issue by Yelinz** [projectcaluma/ember-caluma#2835]
Improve keyboard navigation experience in forms.

It is very difficult to `tab` through a form, to quickly fill out the fields. It focuses the info buttons, only one of the radio buttons, etc.

**Comment by czosel** [comment]
If we tackle this it might be worth reading up on accessibility best practices regarding `tab` navigation. As far as I remember, the goal should be that the application is usable without using a mouse - which would mean that it should be possible to focus (and open) info buttons as well.

**Comment by Venkata-Sai-Vishwanath-robo** [comment]
Hey,

To improve keyboard navigation, WCAG recommends that Tab should focus on the first radio button, with Arrow keys used to navigate between options, and it seems to be working fine in this case. Additionally, WCAG requires a minimum contrast ratio of 4.5:1 between text and background to ensure accessibility.

I also feel that integrating keyboard shortcuts into the form builder could enhance usability further. I’ve recently worked on similar improvements for a project and would be happy to assist if needed!
